settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "[markdown]": {
+        // ...
+        "editor.quickSuggestions": {
+            "other": true,
+            "comments": false,
+            "strings": false
+        }
+        // ...
+    }
```

Comment:
@knzeng-e why this addition of this file?

Reply:
oh good catch, I can remove this file, thanks @jrhender . Its a for a plugin I installed on my vscode to format my md files.